algorithms/arrays/limit.py

```diff
@@ -12,19 +12,25 @@
 Complexity = O(n)
 """
 
-def limit(arr, min_lim = None, max_lim = None):
+
+def limit(arr, min_lim=None, max_lim=None):
     result = []
-    if min_lim == None:
+    if min_lim is None and max_lim:
         for i in arr:
-            if i<= max_lim:
+            if i <= max_lim:
                 result.append(i)
-    elif max_lim == None:
+    elif min_lim and max_lim is None:
         for i in arr:
             if i >= min_lim:
                 result.append(i)
-    else:
+    elif min_lim and max_lim:
         for i in arr:
-            if i >= min_lim and i <= max_lim:
-                result.append(i)
+            if i < min_lim:
+                continue
+            if i > max_lim:
+                continue
```

Comment:
nit
you could probably merge these two ifs into a single if statement.

Reply:
watch you tongue fool

Reply:
im confused